src/components/timeline/TrackContextMenu.js

```diff
@@ -486,7 +486,7 @@ class TimelineTrackContextMenu extends PureComponent<Props> {
     );
   }
 
-  renderHideTrack() {
+  renderHideAllTracksByType() {
```

Comment:
You shouldn't change the `renderHideTrack`, you should add an additional `renderHideAllTracksByType` instead. Can you keep the `hideTrack` and add your new function as additional, please?